src/helpers/steg/forelder.ts

```diff
@@ -96,8 +95,8 @@ export const utfyltBoddSammenAnnenForelder = (forelder: IForelder) => {
 
 export const erIkkeOppgittPgaAnnet = (forelder: IForelder) => {
   return (
-    forelder.hvorforIkkeOppgi?.verdi === EHvorforIkkeOppgi.Annet ||
-    forelder.hvorforIkkeOppgi?.verdi === EHvorforIkkeOppgi.Other
+    forelder.hvorforIkkeOppgi?.verdi === 'Annet' ||
+    forelder.hvorforIkkeOppgi?.verdi === 'Other'
```

Comment:
Jeg liker ikke helt denne strategien, egentlig, for det blir veldig skjørt å sjekke på denne måten.

Reply:
Enig.